generator-fraudpreventiondata/generators/templates/hmrc/README.md

```diff
@@ -0,0 +1,5 @@
+# Title
```

Comment:
I would probably just call this a `helloworld` template. Since that is what it is. So path `generators/templates/helloworld`

Reply:
👍